src/appleseed/foundation/image/genericimagefilewriter.cpp

```diff
@@ -419,7 +419,7 @@ void GenericImageFileWriter::write_scanlines(const size_t image_index)
         }
 
         // Compute y dimensional scanline border.
-        const size_t y_begin = tile_y;
+        const size_t y_begin = tile_y * props.m_tile_height;
```

Comment:
That looked like a serious bug. This function wasn't used anywhere until now?

Reply:
I can't comment there, but there appears to be another bug on line 388:

    std::unique_ptr<uint8> buffer(new uint8[props.m_canvas_width * props.m_tile_height * props.m_pixel_size]);

should be

    std::unique_ptr<uint8[]> buffer(new uint8[props.m_canvas_width * props.m_tile_height * props.m_pixel_size]);